src/omnibar/omnibar.ts

```diff
@@ -554,6 +570,26 @@ function buildOmnibarUrl(): string {
   return omnibarUrl;
 }
 
+function updateTitle(): void {
+  let title = '';
+  let titleParts: string[] = [];
+  if (unreadNotificationCount && unreadNotificationCount !== 0) {
+    titleParts.push(`(${unreadNotificationCount})`);
+  }
+
+  if (currentTitleParts && currentTitleParts.length !== 0) {
+    titleParts = titleParts.concat(currentTitleParts);
+  }
+
+  titleParts.push(serviceName);
+
+  title = titleParts.join(' - ');
```

Comment:
Since the unread notification count is part of the `titleParts` array, wouldn't this put a hyphen in between the unread notification count and the title (e.g. "(2) - Some Title - Some Service")? Typical examples of this behavior on other websites is to not have a hyphen, and that pattern should be followed here (e.g. "(2) Some Title - Some Service").